Traceback (most recent call last):
  File "/projects/bgmp/agwel/bioinfo/Bi622/Demultiplex/Assignment-the-third/demultiplex.py", line 77, in <module>
    with open(args.known_index, 'r') as input: 
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'known_indexes'
Command exited with non-zero status 1
	Command being timed: "/projects/bgmp/agwel/bioinfo/Bi622/Demultiplex/Assignment-the-third/demultiplex.py -r1 read1 -r2 index1 -r3 index2 -r4 read2 -i known_indexes"
	User time (seconds): 0.01
	System time (seconds): 0.01
	Percent of CPU this job got: 39%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:00.07
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 12660
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 2565
	Voluntary context switches: 83
	Involuntary context switches: 0
	Swaps: 0
	File system inputs: 0
	File system outputs: 0
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
